Dockerfile

```diff
@@ -65,7 +65,7 @@ RUN apt-get install -y --no-install-recommends \
 	${with_arm32_as:+binutils-arm-linux-gnueabi} \
 	${with_ppc_as:+binutils-powerpc64le-linux-gnu}
```

Comment:
Can we install `ninja-build` with apt instead? That should be the correct way for debian.

Reply:
Okay 202cb29fa14b19207ddfc0a3632bcbe09bd5e79f